src/app/components/Tokens/TokenList.tsx

```diff
@@ -15,12 +25,40 @@ type TokensProps = {
   pagination: false | TablePaginationProps
 }
 
+export const TokenTypeTag: FC<{ tokenType: EvmTokenType; sx?: SxProps }> = ({ tokenType, sx = {} }) => {
```

Comment:
This is similar to `TokenPills` that wraps around mui's `Chip`.

We should choose more consistent names for "the background rounded label thing"

Reply:
How about `DogTag` ?

Reply:
(I mean for a common ancestor, with common sizing and fond settings, and then create or descendants from that, with consistent names, either Tag or Pill)